irony-server.rst

```diff
@@ -0,0 +1,104 @@
+============
+irony-server
+============
+
+----------------------------------------
+The back-end server for Emacs irony-mode
+----------------------------------------
+
+:Author: Nicholas D Steeves <nsteeves@gmail.com>
+:Date: 2017-10-01
+:Version: 1.1.0
```

Comment:
Sorry, I just updated the version this evening, can you put 1.2.0?

Reply:
Of course! I'll also mark it as a fixup

Reply:
Euh, or not..because it might cause a conflict